Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

atiqueahmedziad opened the pull request:
Added aria-hidden="true" inside template in line 287

jryans committed:
Bug 1127004 - Add WiFi debug auth dialog. r=janx

stasm committed:
Bug 1078187 - Use the shared mock_l10n in Collections. r=kgrandon f=julienw

julienw committed:
Bug 1087981 - [Message] Lazy init settings in notify.js r=schung

mancas committed:
Bug 1105043 - [SHB] SHB does NOT disappear on lockscreen when pressed from callscreen. r=alive

Commit:
Bug 1069614 - making albums accessible in the main view

eeejay committed:
Bug 1068973 - Put headers above, and footers below thumbnails.

Commit:
Bug 864274 - Show battery percentage while charging on lockscreen

mwargers committed:
Bug 1116566 - Create test to catch regression in bug 1116087

FunkTron committed:
Bug 1069630 - Updated "album rating bar" and "caption/share bar" with ARIA roles/labels

Standard8 committed:
Bug 1118444 - Check against all navigator.languages values for possible supported locales. r=stas

sfoster committed:
Bug 1118390 - Lock orientation when task manager shows. r=etienne

Commit:
Bug 1119544 - Remove the unnecessary l10n code from FMD

davehunt committed:
Bug 1074117 - Use expected conditions and optimise element lookups in settings app object. r=fstrugariu

Commit:
Bug 1105304 - [l10n] Privacy Panel: long string truncated, untranslated elements, strings reused f=gandalf r=kgrandon

Commit:
Bug 1106879 - [Lockscreen][RTL] Camera icon with passcode screen is located on the right, causing the time display shifted to the left

Commit:
Bug 961572 - [Messages][Refactoring] Implement a new independent cancel button for information view's header r= Steve Chung

Commit:
Bug 1116803 - [rtl] [contacts] main two contacts app UIs: Contacts List and Add new contact page lack RTL support

 Author:    Ahmed Nefzaoui <nefzaoui.ahmed@hotmail.com>
 Date:      Mon Jan 12 12:31:05 2015 -0800

Commit:
Bug 1113016 - Add a panel to manage Addons

Commit:
Bug 1120878 - [Contacts] View vcard activity should disable edit button if the vcard is an attachment that is already on the detvice

mikehenrty committed:
Bug 1112534 - Move system dialog above gesture panels in z-index

gmarty committed:
Bug 1112706 - Minimized and maximized statusbar can become out of sync

EragonJ committed:
Bug 1107696 - Fix not-in-range problem in Wifi panel, r=crh0716

Right now we can decide whether we should show not-in-range wordings for
not in hidden_network and wifi_list. And also, in hidden_network panel,
we can show some useful information on connected AP now.

cctuan committed:
Bug 1092894 - [System2] Implement NfcCore

Commit:
Bug 1120357 - Show app install/uninstall dialog with Smart Buttons.

mpizza committed:
Bug 1123164 - [Settings]There is no SIM settings icons when you insert SIM2 only in Settings -> Cellular & Data. r=arthurcc.

EragonJ committed:
Bug 1123188 - [Flame][Dialer]It can't enter corresponding settings page
when you tap "Voicemail" and "Fixed Dialing Numbers".

sean2449 committed:
Bug 1122421 [Stingray] UI-Transition - Pin card to home

timdream committed:
Bug 1029951 - Part I, Generate layouts.json instead

timdream committed:
Bug 1029951 - Part II, Let InputMethodDatabaseLoader loads data from IndexedDB

sean2449 committed:
Bug 1122423 [Stingray] UI-Transition - Deck launcher waving effect

tefn3849 committed:
Bug 1114453 - [Wifi] Add wifi setting UX and gaia setting for Allow Offf function.

tefn3849 committed:
Bug 1114453 - [Wifi] Add wifi setting UX and gaia setting for Allow Offf function.

phoenix1moz committed:
Bug 1123548 - Intermittent test_homescreen_column_layout.py

delapuente committed:
Bug 1121586 - [Dialer][Text Selection] Hitting the Delete button will remove the digit from the end of a string of entered digits instead of from where the cursor is currently located.

evanxd committed:
Bug 1120346 - Interropt the event only when the search window has a front window, r=alive

millermedeiros committed:
Bug 1117713 - [Calendar] Disable "All day" when creating an event,tap Save, it will pop up alert message "Something is wrong.Try again later."

eeejay committed:
Bug 1068984 - Assign toolbar/button roles to thumbnail footers and label buttons.

eeejay committed:
Bug 1069007 - Label edit view toolbar buttons and make auto-enhance a toggle button.

eeejay committed:
Bug 1069013 - Make crop and effect buttons aria radio buttons and label them.

eeejay committed:
Bug 1068980 - Moved the fullscreen back button above the image frames tag.

YangYangSPRD committed:
Bug 1110660 - SIM2's PIN1 code will not pop up if SIM1 is not locked

cctuan committed:
Bug 1119704 - blur last module only when latest module is handled, r=alive

daleharvey committed:
Bug 1119820 - Rework configuration of search providers. r=kgrandon

rexboy7 committed:
Bug 1123608 - [Stingray][Notification] Sender App initial commit

Commit:
Bug 1121815 - Interpolate templates by setting data-l10n-id in tv_apps

gpgreen committed:
Bug 819744 - Delay the light sensor response in screen_manager.js r=timdream

gabrielesvelto committed:
Bug 980387 - Remove all remaining uses of navigator.mozMobileConnection r=timdream

RudyLu committed:
Bug 1123250 - Re-organize the lib files used for keyboard integration
 tests.
 - Reland Bug 1123250
 - This reverts commit d998d6bfe3138d793e356908403e323b560b2d19.

timdream committed:
Bug 1029951 - Part IV, UI for layout downloads

timdream committed:
Bug 1029951 - Part III, Model logic for managing dictionaries and layouts

timdream committed:
Bug 1029951 - Part V, Amend CONFIGURE

mancas committed:
Bug 1088723 - Status bar visible while handling pick activity with Camera

mancas committed:
Bug 1119917 - Use of different hashes for Settings activities.r=arthur.chen

alivedise committed:
Bug 1124816 - Focus window if it is top most

mancas committed:
Bug 1121704 - [Contacts] - Contact pick option (select from multiple contact methods) has repeat options when tapping on a contact multiple times before the list loads.

Commit:
Bug 1111981 - [v2.2] Investigate intermittent failure in test_sms_with_picture_attached.py and test_add_photo_to_contact.py

yzen committed:
Bug 1068700 - improved accessibility of email list bottom toolbar. r=jrburke

Commit:
Bug 1059017 - [Calendar] RTL settings will breaks all views

ferjm committed:
Bug 1113551 - [Woodduck][First Time Experience]Can't create a firefox account via FTU. r=sfoster

daleharvey committed:
Bug 1117970 - Pick Search provider from search app

cctuan committed:
Bug 1119731 - correct callback for Commander, r=ricky

gasolin committed:
Bug 1115187 - [RTL] Make input text align right

RudyLu committed:
Bug 1121274 - Intermittent uninstallation_test.js | Show Keyboard App
after uninstallation "before each" hook.
 - Wait for the 2nd input window to present and then switch to that
   frame.

Commit:
Bug 1125037 - [Stingray] Protect _cardList from async external access by AsyncSemaphore

azasypkin committed:
Bug 1124758 - [Messages] We should not add the hidden (+0) text at the end of the header. r=julien

Cwiiis committed:
Bug 1115326 - Don't allow entering edit mode when launching a collection. r=kgrandon

Commit:
Bug 949152 - [FTU] Can't select open hidden wifi (r=sfoster)

azasypkin committed:
Bug 1104961 - Intermittent share_activity_test.js | Messages as share target Share via Messages Activity close button Should return to Thread panel if in Participants panel. Do not use elements that maybe inside activity that is being disposed. r=schung

yzen committed:
Bug 1068705 - improved accessibility of the edit toolbar of the message list card.

albertopq committed:
Bug 1118025 - Only recalculating statusbar width when collapsed

eliperelman committed:
Bug 1115134 - Bumping gaia-raptor

KevinGrandon committed:
Bug 1126494 - Disable theme-color when in private browsing r=etienne

eeejay committed:
Bug 1068998 - Label video player buttons. r=djf

timdream committed:
Bug 1126112 - Remove keyboard.current from Keyboard app

birtles committed:
Bug 1118361 - Detect end of status bar fade using transition events instead of comparing computed style; r=cwiiis

Aspirar committed:
Bug 930871 - Fix CDMA data terminology r=arthurcc

rexboy7 committed:
Bug 1126666 - [System] Support remote opening app/webpage for presentation API

crh0716 committed:
Bug 1115173 - Use the native back button of gaia header

stasm committed:
Bug 1126247 - Hide the Get More Languages link until Marketplace supports langpacks. r=arthurcc

weilonge committed:
Bug 1126691 - Remove STK_EVENT_TYPE_USER_ACTIVITY and STK_EVENT_TYPE_LANGUAGE_SELECTION handlers before registering new events.

EragonJ committed:
Bug 1115211 - fix overlapping problem by udpating lists.css

albertopq committed:
Bug 1115172 - Utility tray date overlays statusbar icons in RTL

Commit:
Bug 1125753 - Fix Gaia Keyboard Build Integration Tests by fixing "expected" data

- Fix "expected" dictionaries.json by amending en-Africa, wo, th layout information into the data structure.

crh0716 committed:
Bug 1126708 - Use dialog service for the enter certificate name dialog

Cwiiis committed:
Bug 1115326 - Test that edit mode can't be entered when launching a collection. r=kgrandon

janjongboom committed:
Bug 1126012 - Remove dev_apps/demo-keyboard

Commit:
Bug 1126404 - Fix ATT APN settings when syncing AOSP APN list. r=jaoo

lissyx committed:
Bug 1119290 - Make LogShake use NotificationsSystemMessage service r=mhenretty,janx

LogShake notifications should be able to trigger their actions after a
reboot. Let's make use of the NotificationsSystemMessage service to
handle this.

Cwiiis committed:
Bug 1120799 - Don't leave space when dragging expanded group. r=kgrandon

gabrielesvelto committed:
Bug 1126845 - Make the MMI response field behave correctly for RTL locales r=drs

ferjm committed:
Bug 1124114 - TrustedWindow does not notify about the dialog being closed by the user. r=alive

KevinGrandon committed:
Bug 1126953 - Autolander test bug, remove strings for quick settings dialog r=kgrandon

tamarahills committed:
Bug 1102344 - Make Call Counts show correctly in the rtl language.

benfrancis committed:
Bug 1117967 - Show a single line of combined local apps, bookmarks and Marketplace results

sfoster committed:
Bug 1010615 - Fix RTL FTU toggle-switches. r=fcampo a=bajaj

sfoster committed:
Bug 1010615 - Fix RTL FTU toggle-switches

lightsofapollo committed:
Bug 1127051 - Switch to docker registry a=bustage r=kgrandon

Commit:
Bug 1109820 - Restart calendar when timezone changes

yzen committed:
Bug 1068703 - improve message list accessibility in edit mode.

jobara committed:
Bug 1068691 - set aria role and label for close

Set the aria role (button) and label for the close button on the message-list edit view.

mikehenrty committed:
Bug 1121349 - Update statusbar icon visibility on sim active state change +autoland

sean2449 committed:
Bug 1122432 [Stingray] UI-Transition - Banner

Commit:
Bug 1102834 - Save UserDictionary TST blob in KeyboardSettingsApp

- An empty word list saves into indexedDB as "undefined" in TST blob
- Add WordListConverter which is the JavaScript version of xml2dict.py
-- More information: https://github.com/mnjul/gaia-kb-dict-tools/tree/release/gaiafication
-- Unit testing of WordListConverter is blackbox testing of the result it converts to.
- Change UserDictionary._saveDict() unit test script such that testing queued saves is more realistic and more streamlined.
- Implement PromiseStorage.setItems()

crh0716 committed:
Bug 1118366 - Keep the direction of the carats in RTL mode

chenpighead committed:
Bug 1120358 - Part2: Gaia JS integration test case v3

Commit:
Bug 1127210 - Remove the Gb checking for mobizilla_default_search.json

gasolin committed:
Bug 1126685 - use attributechanged to monitor v2 BT stat

cctuan committed:
Bug 1127227 - remove dup register for isSendFileQueueEmpty

ian-liu committed:
Bug 1102798 - [Bluetooth] Implement pairing flow in Bluetooth app to support BT v2 API, r=arthur

ochameau committed:
Bug 1043953 - Revert useless package.json in apps/browser. r=rickychien

phoenix1moz committed:
Bug 1127266 - Update test_rocketbar_search.py according to changes in Bug 1117770

Commit:
Bug 1100893 - [Contacts][RTL] Switch for facebook sync is wrong in RTL languages r=sergi

diox committed:
Bug 1126591 - Update Marketplace app package. r=julienw

huchengtw-moz committed:
Bug 1125780 - [Stingray] Use event name to check if we need to invoke _closing handler

Commit:
Bug 1108708 - change padding for RTL in import contacts page

millermedeiros committed:
Bug 1059017 - [Calendar] RTL fixes for week/day views

mcav committed:
[Clock] Double-tapping "Save" on the Alarm Edit screen results in undeletable alarms.

Commit:
Bug 966228 - Email notification with no subject displays subject as 'null'

eeejay committed:
Bug 1069002 - Assign role and label to image edit preview area.

EragonJ committed:
Bug 1127257 - fix subpanels transition direction for RTL

huchengtw-moz committed:
Bug 1127657 - [Stingray] clear and rollback animation state while animation is prepared or playing.

Anindya-Pandey committed:
Bug 796486 - Fix for basicLayoutKey label for most nonabc languages r=timdream

Bug 796486 - Add basicLayoutKey for Korean

ian-liu committed:
Bug 1127707 - [Bluetooth][Text Selection] Follow text selection pattern: pairing dialog should not be selected r=gduan

ian-liu committed:
Bug 1127715 - [Bluetooth][Text Selection] Try to transfer file via bluetooth, the bluetooth selected page still can use copy-pasty function

huchengtw-moz committed:
Bug 1127745 - [Stingray][demo] fling-player listens click event

Commit:
Bug 1127746 - [Stingray] Use navigator.mozPresentation instead of navigator.presentation

huchengtw-moz committed:
Bug 1127749 - [Stingray][demo] fling-player launch itself while it is hidden

huchengtw-moz committed:
Bug 1127755 - [Stingray][demo] apply building blocks to fling-sender

phoenix1moz committed:
Bug 1127762 - Fix intermittent failure in test_music_change_rating.py

cctuan committed:
Bug 1127768 - [Build] Able to run single module in node

gmarty committed:
Bug 1126209 - [RTL][Homescreen]The confirm information is not right-aligned when deleting apps from the homescreen

delapuente committed:
Bug 1127722 - [Usage][Text Selection] When first time use Usage app, description on "Data usage limit" page can be selected

Commit:
Bug 1058790 - [RTL] International numbers are not correctly handled in Contacts r=jmcf

phoenix1moz committed:
Bug 1127856 - Update test_gallery_switch_to_camera.py according to changes in Bug 1068973

millermedeiros committed:
Bug 1119688 - [TextSelection] Calendar event view cursors not adjustable

Commit:
Bug 1107431 - [FTU] Always 'connecting' status on hidden network (r=sfoster)

KevinGrandon committed:
Bug 1127939 - Disable intermittent failing test, TEST-UNEXPECTED-FAIL | apps/search/test/marionette/suggestions_test.js | Search - Suggestions Test Test suggestions

Commit:
Bump gaia node modules to higher rev to get es6-polyfill

Commit:
Bug 1121814 - Upgrade tv apps to use NotificationHelper

eliperelman committed:
Bug 1128180 - Fixing erroneous return in user timing shim

davehunt committed:
Bug 1118697 - Only append a handler for webapps if we are running a local http server. r=me a=test-only

ian-liu committed:
Revert "Bug 1127715 - [Bluetooth][Text Selection] Try to transfer file via bluetooth, the bluetooth selected page still can use copy-pasty function"

daleharvey committed:
Bug 1126025 - Remove unused tests. r=benfrancis

dholbert committed:
Bug 1127898: Drop unnecessary 'height on .composer-button-container. r=azasypkin

The .composer-button-container element will already stretch its height to match its parent, due to the (default) "align-self:stretch" behavior.

cctuan committed:
Bug 1127584 - extend timeout for build test

huchengtw-moz committed:
Bug 1128384 - [Stingray][demo] use string and use onmessage

EragonJ committed:
Bug 1103810 - make 4 call forwarding options shown as dialog

alivedise committed:
Bug 1124216 - Support value selector in searchWindow

evelynhung committed:
Bug 1123564 - initial work of device deck

cctuan committed:
Bug 1127207 - separate collaspe and selection handler and remove touchcarettap event handler (bug 1120750) and resize handler (bug 1119126)

ian-liu committed:
Bug 1127715 - [Bluetooth][File Transfer] Follow text selection pattern, the page of file transfer should not be selected, r=gdaun

huchengtw-moz committed:
Bug 1128438 - [Stingray][demo] add close button to fling sender app for testing

Commit:
Bug 1124989 - [Stingray][Demo] Use string format and update sample messages

EragonJ committed:
Bug 1127294 - fix bluetooth icon position

cctuan committed:
Bug 1125026 - show/hide frontest screenshot, r=alive

etiennesegonzac committed:
Bug 874368 - Improving the perceived performance of notification swipes r=cwiiis
* prevent the constant reflow while dragging
* take speed into account to detect swipes

julienw committed:
Bug 1128483 - Disable intermittent failing test: TEST-UNEXPECTED-FAIL | apps/calendar/test/marionette/month_view_test.js | month view should gray out numbers outside current month

Unfortunately February 2015 has no gray day.

KevinGrandon committed:
Bug 1128275 - Disable intermittent test in calendar/test/unit/day_observer_test.js r=kgrandon

mcav committed:
Bug 886534 - Contend with INTERNALDATE timezones for date-range resyncing. r=asuth r=mcav

KevinGrandon committed:
Bug 1125684 - Update README.md for autolander launch r=kgrandon

ferjm committed:
Bug 1128580 - We notify about the contacts app being interactive before we should. r=francisco

punamdahiya committed:
Bug 1127437 - [Gallery] - Remove extra margin above group headers r=djf

Commit:
Bug 1127975 - Retry marionette tests on failure to rule out harness failures

tamarahills committed:
Bug 1126850 - Fix Conference Call parenthesis in RTL. r=gsvelto

mwargers committed:
Bug 1116175 - Write a gaia UI test for moztrap test: Share a music as a ringtone

crh0716 committed:
Bug 1115268 - Use lazy loader to generate the template r=eragonj

Commit:
Revert "Bug 1125780 - [Stingray] Use event name to check if we need to invoke _closing handler"

This reverts commit 820a9649967c3703f38dfdb48268dc62e29a99b5.

Commit:
Bug 1112986 - [Stingray][Home] Combine AppBookmark with Application

huchengtw-moz committed:
Bug 1128437 - [Stingray][demo] handle session close in fling-player

RudyLu committed:
Bug 1113510 - Intermittent switching_test.js | switch Keyboard App
Checking the switching IME function is available.
 - Wait for the 2nd input window to show up and then switch to it.

Commit:
Bug 1128830 - [Demo][Stingray] Hide dlnaPlayer and remove Notification Sender from TV build

etiennesegonzac committed:
Bug 1128672 - Make the TouchForwarder dipatch mouse events again r=alive

Commit:
Use true/false as state value

tedders1 committed:
Bug-1105035: Allow use of utility tray with full-screen element. r=kgrandon

jfkthame committed:
Bug 1128981 - [Usage] Fix display of widget text in RTL languages r=kgrandon

This corrects the alignment of the widget text in RTL languages, and also avoids clipping the second line for languages (such as Arabic, but likely applies to at least some Indian scripts as well) where the glyphs have a larger vertical extent.

eliperelman committed:
Bug 1128605 - Bumping gaia-raptor

mwargers committed:
Bug 1074117 - Use expected conditions where appropriate and optimise element lookups in system app. r=dhunt

huchengtw-moz committed:
Bug 1125780 - [Stingray] Use event name to check if we need to invoke _closing handler

jobara committed:
Bug 1127955 - converted text to aria-label

Converted the past-other-month and future-other-month to be aria-labels instead of text nodes.

asutherland committed:
Bug 1128285 - [email/IMAP] downloadBodies snippet fetch requests stalling out until connection gets idle-killed. r=mcav

jrburke committed:
Bug 1127160 - [text selection]No new line copied after copying multiple lines from email compose view

crh0716 committed:
Bug 1118671 - Fix the incorrect selector

alivedise committed:
Bug 1120541 - Do not maximize titlebar if it is fatal error

ian-liu committed:
Bug 1122365 - [Bluetooth][Settings] According to Bluetooth class of device to implement icon style. r=arthur

Commit:
Modify notification message as UX suggested, and we are able to use addEventListner on 'message' now

crh0716 committed:
Revert "Merge pull request #27583 from tefn3849/wifi_1114453"

This reverts commit eb824cc6e228a86e7e2e8ce3d3817496f91be8ef, reversing
changes made to 45475198737a504d81932a9c90002902054fce23.

timdream committed:
Bug 1029951 - Part VI, Style nit per visual review

alivedise committed:
Bug 1129329 - Skip inactive module while broadcasting

Commit:
Bug 1128844 - Do not throw error when keyboard-layouts cannot find application

rexboy7 committed:
Bug 1128840 - [Stingray][Home] Polish card list and animation style
               - Avoid trembling of scaling animation in edit mode
               - Add more left margin space before first card
               - Style fine-tuning (on animation time and some colors)

phoenix1moz committed:
Bug 1129016 - Investigate failure in test_settings_do_not_track.py

phoenix1moz committed:
Bug 1129016 - Investigate failure in test_settings_do_not_track.py

timdream committed:
Revert "Merge pull request #25305 from timdream/value-selector-focus"

This reverts commit 47faa4651f8feb11152ef475850cc9641fd5cbeb, reversing
changes made to 4c7e1e5f68ec57241d488568080732b318d652f5.

Conflicts:
	build/jshint/xfail.list

Commit:
Bug 1127216 - [Demo][Stingray] Fix left/right padding of #app-deck-grid-view

Commit:
Bug 1129238 - Truncated text on call barring (r=arthurcc)

Commit:
Bug 1129030 - Xfail TestKill_test_kill_multiple because of Bug 1125759

julienw committed:
Bug 1124141 - The langpack changes break the "load SMS in Nightly" approach - "NotInApp" error r=gandalf

yzen committed:
Bug 1068699 - for email item in the message list making its reading order more sensible for the screen reader.

phoenix1moz committed:
Revert "Bug 1129016 - Investigate failure in test_settings_do_not_track.py"

This reverts commit 591d5a4a79e1d36cc795ddce4701c567a6609f87.

janjongboom committed:
Bug 1121274 - Blur the input field before deleting the IME

Cwiiis committed:
Bug 1120624 - Expand group when adding new collection. r=kgrandon

phoenix1moz committed:
Bug 1129016 - Investigate failure in test_settings_do_not_track.py

phoenix1moz committed:
Bug 1129475 - Fix failure in test_sms_notification.py. r=dhunt

Cwiiis committed:
Bug 1120624 - Fix the offset when long-pressing the homescreen. r=kgrandon

KevinGrandon committed:
Revert "Revert "Merge pull request #25305 from timdream/value-selector-focus""

This reverts commit a785391616adb07777dd0e94c8bd52d432d8fdd9.

Cwiiis committed:
Bug 1093270 - Increase hit target for group headers. r=kgrandon

Commit:
Bug 1128483 - Fix month view test that checks for greyed-out days outside current month

mikehenrty committed:
Revert "Merge pull request #25291 from anrao91/840790"

This reverts commit 763683f27482075caaed47f9348e84ebf8a175d5, reversing
changes made to 12e2da14efee4fa564759ef64038b6c6cfe4a04b.

Commit:
Bug 1129563 - use window.location.hash instead of mozHasPendingMessages()

qdot committed:
Bug 1115130 - Make building for raptor turn on user timing logging preference

sfoster committed:
Bug 1128630 - Browser windows should show URL + SSL state in task manager/tab view

Commit:
Bug 1128424 - Stop using deck images in smart-home and remove 'cachedIconURL'

mikehenrty committed:
Bug 1125962 - Reset SIM PIN form after each entry attempt

huchengtw-moz committed:
Bug 1129807 - [Stingray][demo] hide and release video while visibility is hidden

Commit:
Bug 808624 - [radio] [UX] favorites star icon may go unnoticed. Consider make it more visible

timdream committed:
Bug 1129822 - Print node/npm versions

Commit:
Add @fileoverview to gaia_marionette_retry.js a=test-only

daleharvey committed:
Bug 1127939 - Reenable suggestions test

Commit:
Bug 1125798 - Investigate camera switch source failure on automation

eliperelman committed:
Bug 1129273 - Bumping gaia-raptor r=kgrandon

phoenix1moz committed:
Bug 1128952 - Use more precise actions when selecting values from the timepicker

etiennesegonzac committed:
Bug 1130011 - Removing superfluous touch event listener on the whole window r=kgrandon

sfoster committed:
Revert "Bug 1010615 - Fix RTL FTU toggle-switches" for bug 1129922

This reverts commit 7a607eb3ebfc9ed75f388d3b3b2f7f41a43c9867.

tamarahills committed:
Bug 1126379 - Force LTR for phone number and delete in the Emergency Dialer.

KevinGrandon committed:
Bug 1092022 - Disable intermittent failing test, TEST-UNEXPECTED-FAIL | /builds/slave/test/gaia/apps/system/test/marionette/browser_fullscreen_navigation_chrome_test.js | Browser - App /w Fullscreen Navigation Chrome test progressbar

eliperelman committed:
Bug 1129041 - Calendar: remove user timing shim

Commit:
Bug 1129805 - Remove banner-button's italic style and apply it to all smart-button.

allstarschh committed:
Bug 1129809 - Make the URL in NFC-API-Test app configurable. r=timdream

crh0716 committed:
Bug 1129830 - Change to use native gaia header action

KevinGrandon committed:
Revert "Bug 1127939 - Reenable suggestions test"

This reverts commit 598863b8070d3b74c0a9454cfc44879c20f65b44.

evelynhung committed:
Bug 1130283 - remove DOM_VK_F6 reload shortcut

daleharvey committed:
Bug 1117972 - Show search notice again. r=benfrancis

Commit:
Bug 1130344 - [FM Radio] Update background of Dialog background

Commit:
Bug 1130410 - Remove optimize-clean.js from post-app.js

daleharvey committed:
Revert "Bug 1117970 - Pick Search provider from search app"

This reverts commit ed368bc76f6e2c4ac3817c90494a7b52b2be42d0.

daleharvey committed:
Bug 1127939 - Wait for provider ready before searching. r=kgrandon

rvandermeulen committed:
Revert "Bug 1130011 - Removing superfluous touch event listener on the whole window r=kgrandon"

This reverts commit 33e864db8d1576eeef537c26af89f7a53b85213a.

Commit:
Bug 1129572 - make MediaDB send an enumerable event before it is fully ready

KevinGrandon committed:
Bug 1130552 - Disable broken test, TEST-UNEXPECTED-FAIL | null | [privacy-panel-test/unit/ala/blur_slider_test.js] ALA BlurSlider "before all" hook

Commit:
Bug 1124141 - Update l10n.js to work with pre-ES6 JS

mikehenrty committed:
Bug 1118713 - Absolutely position the progress element for RTL

daleharvey committed:
Bug 1117970 - Reland - Pick Search provider from search app. r=alive, r=benfrancis

This reverts commit 0244121522343877d65a69377226a836688e3004.

mwargers committed:
Revert "Merge pull request #27867 from mwargers/share_music_ringtone2"

This reverts commit 53d81870dc547fa57b915b06457142e4ee1051b2, reversing
changes made to 1ea6c3c6c918fcc76598d3a5bafe07200763bab4.

KevinGrandon committed:
Bug 1106420 - Disable intermittent test, apps/system/test/marionette/media_playback_test.js | media playback tests should hide now playing info by exiting r=kgrandon

Commit:
Follow up to Bug 1115796 - Update l10n.js to use Langpacks API. r=stas

KevinGrandon committed:
Bug 1130865 - Disable permaorange test on Sunday | apps/calendar/test/marionette/month_view_test.js | month view double tap r=kgrandon

EragonJ committed:
Bug 1130307 - Fix RTL problems in download manager

EragonJ committed:
Bug 1130621 - Fix wifi-hotspot security position in RTL languages

atiqueahmedziad committed:
Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

Added aria-hidden="true" inside template in line 287

Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

Added aria-hidden="true" inside template in line 287

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/8364578?v=3" alt="atiqueahmedziad" width="50px" height="50px" /> Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9dace299f9ba)

wilsonpage commented on apps/camera/js/views/viewfinder.js:
```diff
@@ -284,7 +284,7 @@ module.exports = View.extend({
   template: function() {
     return '<div class="viewfinder-frame js-frame">' +
-        '<div class="viewfinder-video-container js-video-container">' +
+        '<div class="viewfinder-video-container js-video-container" aria-hidden="true">' +
```
There's a linter fail: `apps/camera/js/views/viewfinder.js | line 287, col 90, Line is too long.`

Lines can't be longer that 80 characters. Please wrap it :)

wilsonpage replied:
Sorry I should have been more clear:

```js
'<div class="viewfinder-video-container js-video-container" ' +
'aria-hidden="true">' +
```

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/8364578?v=3" alt="atiqueahmedziad" width="50px" height="50px" /> Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=92d2a69c81e8)

atiqueahmedziad commented:
Thanks :) Done :)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/8364578?v=3" alt="atiqueahmedziad" width="50px" height="50px" /> Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7928550e824e)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/8364578?v=3" alt="atiqueahmedziad" width="50px" height="50px" /> Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2664ef85f27a)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/8364578?v=3" alt="atiqueahmedziad" width="50px" height="50px" /> Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d39a9722967a)

atiqueahmedziad committed:
This reverts commit f97dc7afd7cd9a3058abd3e56821c642fabc6653.

Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

Added aria-hidden="true" inside template in line 287

Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

Added aria-hidden="true" inside template in line 287

Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

Added aria-hidden="true" in line 288

Bug 1068874 - [Camera] Focus indicator graphic should be hidden from screen reader

Added aria-hidden="true" in line 288

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/8364578?v=3" alt="atiqueahmedziad" width="50px" height="50px" /> Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=af7480c0a27d)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/8364578?v=3" alt="atiqueahmedziad" width="50px" height="50px" /> Atique Ahmed Ziad (atiqueahmedziad)](https://github.com/atiqueahmedziad) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9fc109d3078b)
